xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -153,19 +158,24 @@ bool CGUIWindowVideoNav::OnMessage(CGUIMessage& message)
             break;
           }
         }
-        if (i >= m_vecItems->Size() && url.GetOption("showinfo") == "true")
+        if (i >= m_vecItems->Size())
         {
-          // We are here if the item is filtered out in the nav window
-          std::string path = message.GetStringParam(0);
-          CFileItem item(path, URIUtils::HasSlashAtEnd(path));
-          if (item.IsVideoDb())
+          SelectFirstUnwatched();
```

Comment:
Above which if/else?
If its `if (message.GetStringParam(0) != "")`, yes, it can, just that it would be executed unnecassirily more often...
If its `if (i >= m_vecItems->Size())` then no, because an item might have been select by the param